packages/components/input/stories/input.stories.tsx

```diff
@@ -527,6 +527,7 @@ export const Required = {
   args: {
     ...defaultProps,
     isRequired: true,
+    validationBehavior: "native",
```

Comment:
I don't think it needs to be `native` as it can be changed in the control panel.
![image](https://github.com/nextui-org/nextui/assets/76232929/1829cc1c-79c1-4fc8-b123-50551ac5dbcb)

Reply:
Yes but most people don't aware of it. I think the main idea for this story is to demonstrate that we can use `isRequired` to prevent the form submission when the input is empty. By default, we are using aria so you can submit it without inputting any value, which may be confusing. That's why I set `native` here for demonstration purpose.